app/src/pages/index.tsx

```diff
@@ -13,6 +13,20 @@ interface PageProps {
   isFooEnabled: boolean;
 }
 
+// Change this to getStaticProps if you're not using server-side rendering
```

Comment:
ℹ️ Just moved `getServerSideProps` to the top of the file instead of bottom. I think this makes it easier to read through a file, more closely following the flow of the data from server to page prop.